worker/workercmd/worker_linux.go

```diff
@@ -43,12 +43,17 @@ type ContainerdRuntime struct {
 	CNIPluginsDir  string        `long:"cni-plugins-dir" default:"/usr/local/concourse/bin" description:"Path to CNI network plugins."`
 	RequestTimeout time.Duration `long:"request-timeout" default:"5m" description:"How long to wait for requests to Containerd to complete. 0 means no timeout."`
 
-	//TODO can DNSConfig be simplifed to just a bool rather than struct with a bool?
-	DNS                DNSConfig `group:"DNS Proxy Configuration" namespace:"dns-proxy"`
-	DNSServers         []string  `long:"dns-server" description:"DNS server IP address to use instead of automatically determined servers. Can be specified multiple times."`
-	RestrictedNetworks []string  `long:"restricted-network" description:"Network ranges to which traffic from containers will be restricted. Can be specified multiple times."`
-	MaxContainers      int       `long:"max-containers" default:"250" description:"Max container capacity. 0 means no limit."`
-	NetworkPool        string    `long:"network-pool" default:"10.80.0.0/16" description:"Network range to use for dynamically allocated container subnets."`
+	Network struct {
+		ExternalIP flag.IP `long:"external-ip" description:"IP address to use to reach container's mapped ports. Autodetected if not specified."`
+		//TODO can DNSConfig be simplifed to just a bool rather than struct with a bool?
+		DNS                DNSConfig `group:"DNS Proxy Configuration" namespace:"dns-proxy"`
+		DNSServers         []string  `long:"dns-server" description:"DNS server IP address to use instead of automatically determined servers. Can be specified multiple times."`
+		RestrictedNetworks []string  `long:"restricted-network" description:"Network ranges to which traffic from containers will be restricted. Can be specified multiple times."`
+		Pool               string    `long:"network-pool" default:"10.80.0.0/16" description:"Network range to use for dynamically allocated container subnets."`
+		MTU                int       `long:"mtu" description:"MTU size for container network interfaces. Defaults to the MTU of the interface used for outbound access by the host. Max allowed value is 1500."`
```

Comment:
While it's true [ethernet v2](https://en.wikipedia.org/wiki/Ethernet_frame#Ethernet_II) is capped at 1500, I don't see any enforcement of this (unless it's at the CNI level).

Reply:
also just copied the help text from Guardian - removed the note